app/Http/Controllers/ImportController.php

```
@@ -526,10 +527,20 @@ private function safelyGetCSVLine($file, $delimiter, $quoteChar)
 
     /**
-     * Check if a transaction already exists in the database
+     * Determines whether a transaction is a duplicate for the given account.
+     *
+     * Currently always returns false, effectively disabling duplicate transaction detection.
+     *
+     * @param  array  $data  Transaction data to check.
+     * @param  mixed  $accountId  Account identifier.
+     * @return bool Always returns false.
      */
     private function isDuplicateTransaction(array $data, $accountId): bool
```

PHPStan (2.1.15)

Method App\Http\Controllers\ImportController::isDuplicateTransaction() has parameter $data with no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)